src/api/core.js

```diff
@@ -137,8 +137,8 @@ export const parseEndpoints = data => {
 				const { group, description } = guessEndpointDocumentation( method, data.namespace, pathLabel );
 
 				const endpoint = {
-					path_format: pathFormat || '/',
-					path_labeled: pathLabel || '/',
+					pathFormat: pathFormat || '/',
+					pathLabeled: pathLabel || '/',
```

Comment:
Yes :( I fixed this in https://github.com/Automattic/wp-api-console/pull/32/commits/034284d17a9a2f1cdf7e0ba023064ffd0dac8aa1.